Package.swift

```diff
@@ -3,12 +3,6 @@ import PackageDescription
 
 let package = Package(
     name: "Sentry",
-    platforms: [
-        .macOS(.v10_10),
-        .iOS(.v8),
-        .tvOS(.v9),
-        .watchOS(.v2)
-    ],
```

Comment:
`h`: Please don't remove the platforms here. Can we just change it to `.iOS(.v9)`

Reply:
Hey! I’m happy to make this change.

However. 

The reason I removed the explicit deployment targets is this will retain the previous behavior of having iOS 8 as the deployment target for Xcode 11.

I figured in a years fine when iOS 9 is possibly removed from Xcode 13, we will need to do a similar PR to change the minimum deployment target to iOS 10.

Omitting the target all together will do this automatically and implicitly.

Same is true of watchOS, tvOS, and macOS.

What do you think is the better approach?
I personally advocate for removing the explicit versioning since there is no practical reason for it in Sentry’s case.